### ISSUE TITLE
Macos 12 fails to install

### ISSUE THREAD

**Issue by RyzenDew** [notAperson535/OneClick-macOS-Simple-KVM#19]
Tried to install macos 12 but as you can see it failed
Tried master and preview both failed!

![image](https://user-images.githubusercontent.com/84041391/174227242-b7e174ac-9ee9-4cb6-95be-73babc562d64.png)

Fedora 36

**Comment by Tadjaur** [comment]
+1
I have the same issue

**Comment by notAperson535** [comment]
@RyzenDew @Tadjaur First, do not use the preview branch. Second, did you format as macOS extended journaled?

**Comment by RyzenDew** [comment]
> @RyzenDew @Tadjaur First, do not use the preview branch. Second, did you format as macOS extended journaled?

happens with master also and i did format as required it just did not work

**Comment by notAperson535** [comment]
@Tadjaur @RyzenDew @messi142857 try Big Sur and see if it has the same issue. Monterey mostly has features that would be useless in vmd

**Comment by messi142857** [comment]
@notAperson535 I have an RX 6600XT GPU which, from what I gather, only works on macOS Monterey.

**Comment by messi142857** [comment]
I have the exact same error (something to do with bootbase.efi), and I'm running Arch (btw).

**Comment by RyzenDew** [comment]
Happens on arch linux also and popos Sent from my iPhoneOn Jun 22, 2022, at 4:33 PM, notAperson535 ***@***.***> wrote:﻿
And you all have the exact same error? And you are all running Fedora 36 or just Fedora

**Comment by ErichPetersen** [comment]
Same on Ubuntu 22.04.

**Comment by notAperson535** [comment]
@RyzenDew Please rerun the installer and show all logs, not just errors.

**Comment by messi142857** [comment]
At the risk of sounding like I'm hijacking this thread, here's my installer log.
[Installer Log 24-Jun-2022.txt](https://github.com/notAperson535/OneClick-macOS-Simple-KVM/files/8977709/Installer.Log.24-Jun-2022.txt)

I was free, so I thought I might as well provide it.

**Comment by notAperson535** [comment]
@Tadjaur @RyzenDew @messi142857 Are you all using and ryzen processors?

**Comment by RyzenDew** [comment]
I am but that shouldn’t be an issue due to qemu/kvm spoofing the cpu like what we do with unraid unless something was broken and not done right in opencore

**Comment by messi142857** [comment]
> @Tadjaur @RyzenDew @messi142857 Are you all using and ryzen processors?

Yes. I'm using a Ryzen 5 5600G, if it makes any difference.

**Comment by notAperson535** [comment]
> At the risk of sounding like I'm hijacking this thread, here's my installer log. [Installer Log 24-Jun-2022.txt](https://github.com/notAperson535/OneClick-macOS-Simple-KVM/files/8977709/Installer.Log.24-Jun-2022.txt)
> 
> I was free, so I thought I might as well provide it.

@messi142857 doesn't appear online with any results.

**Comment by messi142857** [comment]
> > At the risk of sounding like I'm hijacking this thread, here's my installer log. [Installer Log 24-Jun-2022.txt](https://github.com/notAperson535/OneClick-macOS-Simple-KVM/files/8977709/Installer.Log.24-Jun-2022.txt)
> > I was free, so I thought I might as well provide it.
> 
> @messi142857 doesn't appear online with any results.

Are you referring to the fact that I wasn't online for the past few days? If so, that's because I had some other work to attend to, but I'll be available if you need me to provide further help.

**Comment by notAperson535** [comment]
> Are you referring to the fact that I wasn't online for the past few days? If so, that's because I had some other work to attend to, but I'll be available if you need me to provide further help.

@messi142857 no that is not what I was referring to. I was saying that when I searched up the error message, I could not find any results.

**Comment by notAperson535** [comment]
@messi142857 @Tadjaur @RyzenDew Try running `git pull --rebase` to update open core

**Comment by messi142857** [comment]
> > Are you referring to the fact that I wasn't online for the past few days? If so, that's because I had some other work to attend to, but I'll be available if you need me to provide further help.
> 
> @messi142857 no that is not what I was referring to. I was saying that when I searched up the error message, I could not find any results.

Oh, my bad then.

I did find a comment under this [this article](https://www.nicksherlock.com/2021/10/installing-macos-12-monterey-on-proxmox-7/) that says this issue only occurs in Monterey 12.4. Would you be able to provide the Monterey 12.3.1 image instead?

Also, running `git pull --rebase` didn't work for me.

**Comment by notAperson535** [comment]
@messi142857  hmm, I will try to find a Monterey 12.3.1 and put it in this thread.

**Comment by notAperson535** [comment]
@messi142857 [see if this works](https://drive.google.com/file/d/1K8YJOOHkCshBXHFLGCPUpher912ToaRC/view?usp=sharing). rename it to base system.dmg and then turn it into base system.img with this command: `qemu-img convert BaseSystem.dmg -O raw BaseSystem.img`

**Comment by messi142857** [comment]
@notAperson535 Thanks for helping. However, I'm getting the error `qemu-img: Could not open 'BaseSystem.dmg': Could not locate UDIF trailer in dmg file` when I try to convert the dmg file into an img.

I may have found another solution, however. I found a Monterey 12.3.1 ISO file [here](https://archive.org/details/macos-collection) that seems to have worked for me. I had to change the basic.sh file to point to it, however, but once I did, I was able to install macOS Monterey.

**Comment by notAperson535** [comment]
@messi142857 Perfect! That works too.

**Comment by notAperson535** [comment]
> @notAperson535 Thanks for helping. However, I'm getting the error `qemu-img: Could not open 'BaseSystem.dmg': Could not locate UDIF trailer in dmg file` when I try to convert the dmg file into an img.
> 
> I may have found another solution, however. I found a Monterey 12.3.1 ISO file [here](https://archive.org/details/macos-collection) that seems to have worked for me. I had to change the basic.sh file to point to it, however, but once I did, I was able to install macOS Monterey.

@RyzenDew @Tadjaur solution ^^^

**Comment by RyzenDew** [comment]
that is not a fix

**Comment by notAperson535** [comment]
@RyzenDew well if you find a better Monterey 12.3.1 image to use than post it here

**Comment by notAperson535** [comment]
@RyzenDew @Tadjaur The new versions of Monterey fix this. I installed it successfully.

**Comment by RyzenDew** [comment]
Thats not a fix .. at all thats a janky work around at best Sent from my iPhoneOn Jul 6, 2022, at 6:13 PM, notAperson535 ***@***.***> wrote:﻿

@notAperson535 Thanks for helping. However, I'm getting the error qemu-img: Could not open 'BaseSystem.dmg': Could not locate UDIF trailer in dmg file when I try to convert the dmg file into an img.
I may have found another solution, however. I found a Monterey 12.3.1 ISO file here that seems to have worked for me. I had to change the basic.sh file to point to it, however, but once I did, I was able to install macOS Monterey.

@RyzenDew @Tadjaur solution ^^^
